src/containers/Review/notes/NotesTab.tsx

```diff
@@ -1,25 +1,221 @@
+                    }}
+                    style={{
+                      visibility: isCurrentUser && canDelete(note.timestamp) ? 'visible' : 'hidden',
+                    }}
+                  />
+                </div>
+                <FilesDisplay files={note.files.nodes as FileData[]} />
+                <div className="note-footer-row">
+                  <p className="tiny-bit-smaller-text">
+                    <strong>{note.user?.fullName}</strong>
+                  </p>
+                  <p className="slightly-smaller-text dark-grey">
+                    {DateTime.fromISO(note.timestamp).toLocaleString(DateTime.DATETIME_MED)}
+                  </p>
+                </div>
+              </div>
+            )
+          })
+        ) : (
+          <Message floating header={strings.REVIEW_NOTES_NO_NOTES} />
+        )}
+        {!showForm && (
+          <div className="item-container">
+            <Form.Field>
+              <Button
+                primary
+                inverted
+                className="wide-button"
+                onClick={() => setState({ ...state, showForm: true })}
+              >
+                <Icon name="plus" size="tiny" color="blue" />
+                {strings.REVIEW_NOTES_NEW_NOTE}
+              </Button>
+            </Form.Field>
+          </div>
+        )}
+      </div>
+      {showForm && (
+        <NewCommentForm
+          structure={fullStructure}
+          state={state}
+          setState={setState}
+          refetchNotes={refetch}
+        />
+      )}
     </Container>
-  )
+  ) : null
 }
 
 export default NotesTab
+
+const FilesDisplay: React.FC<any> = ({ files }) => {
+  if (files.length === 0) return null
+
+  return (
+    <div className="file-row">
+      {files.map((file: FileData) => (
+        <div className="file-container" key={file.uniqueId}>
+          <a href={`${downloadUrl}/file?uid=${file?.uniqueId}`} target="_blank">
+            <Image src={`${downloadUrl}/file?uid=${file?.uniqueId}&thumbnail=true`} />
+          </a>
+          <a href={`${downloadUrl}/file?uid=${file?.uniqueId}`} target="_blank">
+            {file.originalFilename}
+          </a>
+        </div>
+      ))}
+    </div>
+  )
+}
+
+const sortAndFilter = (
+  notes: ApplicationNote[],
+  sortDesc: boolean,
+  filesOnly: boolean
+): ApplicationNote[] => {
+  const sort = sortDesc ? (e: ApplicationNote[]) => e : (e: ApplicationNote[]) => [...e].reverse()
+  const fileFilter = filesOnly
+    ? (e: ApplicationNote[]) => e.filter((note) => note.files.nodes.length > 0)
+    : (e: ApplicationNote[]) => e
+  return fileFilter(sort(notes))
+}
+
+const canDelete = (timestamp: string): boolean => {
+  // This is pretty insecure -- all someone needs to do is change their local
+  // time on their computer and they can delete whatever they want!
+  return (
+    Interval.fromDateTimes(DateTime.fromISO(timestamp), DateTime.now()).length('minutes') <
+    COMMENT_DELETION_LIMIT
+  )
+}
```

Comment:
Have you added an issue to the future? I think it's better to do

Reply:
We should do it, but we haven't decided what the delete policy should be.

Have made an issue to discuss and plan: #1105